frontend/src/metabase/dashboard/components/DashboardSidebars.jsx

```diff
@@ -168,7 +166,6 @@ export function DashboardSidebars({
       return (
         <DashboardInfoSidebar
           dashboard={dashboard}
-          saveDashboardAndCards={saveDashboardAndCards}
```

Comment:
since I connect to redux directly inside `DashboardInfoSidebar` I don't need to pass this prop anymore.